packages/alfa-aria/src/node.ts

```diff
@@ -207,17 +208,9 @@ export namespace Node {
 
     // Find all elements in the tree. As explicit ownership is specified via ID
     // references, it cannot cross shadow or document boundaries.
-    const elements = root.inclusiveDescendants().filter(dom.Element.isElement);
-
-    // Build a map from ID -> element to allow fast resolution of ID references.
-    // The collected references are added to the map in reverse order to ensure
-    // that the first occurrence of a given ID is what ends up in the map in
-    // event of duplicates.
-    const ids = Map.from(
-      elements
-        .collect((element) => element.id.map((id) => [id, element] as const))
-        .reverse()
-    );
+
+    const elements = getElementDescendants(root);
```

Comment:
I don't think we need a special function (especially since it duplicate the cache, …) We can just do `getElementDescendants(element).append(element)` if need be.